SocialAndLocalAccountsWithMfa/TrustFrameworkBase.xml

```diff
@@ -1101,7 +1103,7 @@
           <DisplayName>Session Mananagement Provider</DisplayName>
           <Protocol Name="Proprietary" Handler="Web.TPEngine.SSO.DefaultSSOSessionProvider, Web.TPEngine, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" />
           <PersistedClaims>
-            <PersistedClaim ClaimTypeReferenceId="Verified.strongAuthenticationPhoneNumber" />
```

Comment:
Verified.strongAuthenticationPhoneNumber is not read from the directory and is thus not a valid claim in a sign-in user flow.